src/main/java/core/basesyntax/dao/DataConverterImpl.java

```diff
@@ -0,0 +1,36 @@
+package core.basesyntax.dao;
+
+import core.basesyntax.models.FruitTransaction;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class DataConverterImpl implements DataConverter {
+    private static final String HEADER = "type,product,quantity";
+    private static final String SPLITTER = ",";
+
+    public List<FruitTransaction> convertToTransaction(List<String> productsInString) {
+        List<FruitTransaction> fruitTransactions = new ArrayList<>();
+        for (String productStr : productsInString) {
+            if (productStr.equals(HEADER)) {
+                continue;
+            }
```

Comment:
Let's use `for (int i = HEADER_LINE_INDEX; i < productsInString.size(); i++)` loop and remove `if` statement
```suggestion
        for (int i = HEADER_LINE_INDEX; i < productsInString.size(); i++)
```